pydatastructs/linear_data_structures/algorithms.py

```diff
@@ -1369,3 +1369,82 @@ def bubble_sort(array, **kwargs):
         array._modify(force=True)
 
     return array
+
+
+# Under evaluation currently
+def radix_sort(array, **kwargs):
+    """
+        Performs Radix Sort on given input array
+        Current limitations: Works for integers >= 0
+        Parameters
+        ==========
+
+        array: Array
+            The array which is to be sorted.
+
+        backend: pydatastructs.Backend
+            The backend to be used.
+            Optional, by default, the best available
+            backend is used.
+
+        Returns
```

Comment:
Shift back by one level of indentation.

Reply:
Yes, I have done this and will add in next commit.